content/TOC.md

```diff
@@ -0,0 +1,46 @@
+**Contents**
+
+1. [Introduction](index.md)
+2. [Getting started](getting-started/index.md)
+    1. [Core concepts](getting-started/core-concepts.md)
+    2. [Objectives]()
+    3. [Resources & economy]()
+    4. [Production & expansion]()
+    5. [Rules & etiquette]()
+3. [Interface & Gameplay](interface-gameplay/index.md)
+    1. [Home screen](interface-gameplay/home.md)
+        - [Score Boards](interface-gameplay/scoreboards.md)
+    2. [Planets]()
+        - [Planets list]()
+        - [Building]()
+        - [Ship yard]()
+        - [Barracks]()
+        - [Communications]()
+            - [Radar]()
+            - [Scanning]()
+    3. [Fleets]()
```

Comment:
I am not sure if this belongs under Fleets or Navigation, but we might want to say something \ have a section about Travel Time.

For instance, we have jump gates and beacons, as well as travel from Home vs travel from in system.